pkg/reconciler/v1alpha1/testing/aliases.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2018 The Knative Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testing
```

Comment:
Just curious why we are doing this aliasing instead of using those packages directly?

Reply:
it makes testing easier and gives you a hook to overload things for the version.

Reply:
Ok, I'll talk to you offline so you can educate me about this.

Reply:
Just import the appropriate things?

Reply:
I will do this as a followup when serving changes.

Reply:
done.

Reply:
deleted.